examples/java/spring-boot/build.gradle

```
@@ -16,7 +16,8 @@ dependencies {
 }
 
 jsonSchema2Pojo {
-  source = files("../../../schemas/v1alpha")
+  def fileTree = fileTree(dir:'../../../schemas/v1alpha', excludes:['*.yml'])
```

I would generally recommend allow-list rather than block-list approach. It's more likely we'll be adding other file types into this directory (i.e. `.yaml` as well as `.yml`) that could cause us problems, so instead we can take control of what we know we want, which are JSON files

```suggestion
  def fileTree = fileTree(dir:'../../../schemas/v1alpha', include:['*.json'])
```